README.md

```diff
@@ -73,24 +74,23 @@ docker build --build-arg TF_VERSION=1.15.0-gpu -t $USER/deepcell-tf .
 ### Run the new docker image
 
 ```bash
-# NV_GPU refers to the specific GPU to run DeepCell on, and is not required
-
-NV_GPU='0' nvidia-docker run -it \
+# '"device=0"' refers to the specific GPU(s) to run DeepCell-tf on, and is not required
+docker run --gpus '"device=0"' -it \
 -p 8888:8888 \
-$USER/deepcell-tf:0.4.0-gpu
```

Comment:
Let's leave it as the versioned release for now. `latest` may be unreliable, especially in regards to GPU vs non-GPU dependencies. We can update to 0.5.0 when that release is ready.